stdlib/public/core/SwiftifyImport.swift

```diff
@@ -62,7 +62,8 @@ public enum _SwiftifyInfo {
 @attached(peer, names: overloaded)
 public macro _SwiftifyImport(_ paramInfo: _SwiftifyInfo...,
                              spanAvailability: String? = nil,
-                             typeMappings: [String: String] = [:]) =
+                             typeMappings: [String: String] = [:],
+                             debug: Bool = false) =
```

Comment:
@DougGregor do you think including conditionally enableable debug logging in the macro is reasonable, or is that a no-no in the stdlib?